liquibase-core/src/main/java/liquibase/change/AbstractChange.java

```diff
@@ -411,6 +415,57 @@ public CheckSum generateCheckSum() {
         return CheckSum.compute(new StringChangeLogSerializer().serialize(this, false));
     }
 
+    /**
+     * Gets sql content from provided file
+     *
+     * @param path                      path to sql file
+     * @param encoding                  The encoding of the file containing SQL statements
+     * @param isRelativeToChangelogFile is file path relative to changelog
+     * @return sql content
+     * @throws UnexpectedLiquibaseException in case of problems with reading file
+     */
+    protected String getChangeContent(String path, String encoding, Boolean isRelativeToChangelogFile) {
```

Comment:
I think we should have a more descriptive name than getChangeContent().  Maybe openStreamAndExpandExpressions() ?